### PR TITLE
Modified the mail template

### PR DESCRIPTION
**Pull request by Hrishabh17**
## Summary by CodeRabbit

- **New Features**
	- Enhanced email notifications with dynamic URLs and clearer text content for better user experience.
- **Refactor**
	- Added new environment variable `INTEGRATIONS_APP_URL` to improve integration capabilities.

### CODE REVIEW

File: apps/workspaces/templates/mail_template.html [hunk 1]
```diff
@@ -28,15 +28,15 @@
...
+                    <img src="{{fyle_url}}/app/admin/assets/images/icons/fyle/fyle_logo_black_full.png" alt="Fyle" title="Fyle"
                       style="display:block" height="23" width="67" />
                   </td>
                   <td align="right">
                     <span style="padding-left:8px">+</span>
                   </td>
                   <td align="right">
                     <img
-                      src="https://sage-intacct.fyleapps.tech/assets/images/pngs/sage-intacct-logo.png"
+                      src="https://integrations.fyleapps.tech/assets/logos/intacct-logo.png"
```
Comment (ruuushhh) [inline apps/workspaces/templates/mail_template.html:39]: please add this URL to env variable as well

**Review by ruuushhh — APPROVED** [review]

File: apps/workspaces/templates/mail_template.html [hunk 2]
```diff
@@ -28,15 +28,15 @@
               <table width="35%">
                 <tr>
                   <td>
-                    <img src="https://sage-intacct.fyleapps.tech/assets/images/pngs/fyle-logo.png" alt="Fyle" title="Fyle"
+                    <img src="{{fyle_url}}/app/admin/assets/images/icons/fyle/fyle_logo_black_full.png" alt="Fyle" title="Fyle"
```
Comment (ashwin1111) [inline apps/workspaces/templates/mail_template.html:31]: let's upload this to our integrations-app and use that link. This url may fail any time when there are changes in fyle-app

File: fyle_intacct_api/tests/settings.py [hunk 3]
```diff
@@ -287,6 +287,7 @@
 SI_SENDER_PASSWORD = os.environ.get('SI_SENDER_PASSWORD')
 INTEGRATIONS_SETTINGS_API = os.environ.get('INTEGRATIONS_SETTINGS_API')
 INTACCT_INTEGRATION_APP_URL = os.environ.get('INTACCT_INTEGRATION_APP_URL')
+FYLE_INTEGRATION_URL = os.environ.get('FYLE_INTEGRATION_URL')
```
Comment (ashwin1111) [inline fyle_intacct_api/tests/settings.py:290]: INTEGRATIONS_APP_URL

File: apps/workspaces/templates/mail_template.html [hunk 4]
```diff
@@ -22,21 +22,21 @@
           style="font-family: Helvetica,Arial; border-collapse: collapse; background-color: #ffffff; border-radius: 4px; border: solid 1px #d8d8d8;">
           <tr>
             <td bgcolor="#f3f5f6"
-              background="https://raw.githubusercontent.com/fylein/fyle-netsuite-api/b1ebe6c1db08c4e85cc34244f8bf5a1b3f1219ea/apps/workspaces/templates/images/background.png"
+              background="{{fyle_url}}/assets/logos/email/background.png"
```
Comment (ashwin1111) [inline apps/workspaces/templates/mail_template.html:25]: why is this fyle_url and not integrations_app_url?

File: apps/workspaces/tasks.py [hunk 5]
```diff
@@ -176,8 +176,11 @@ def run_email_notification(workspace_id):
                         'workspace_id': workspace_id,
                         'export_time': export_time.date() if export_time else datetime.now(),
                         'year': date.today().year,
-                        'app_url': "{0}/app/settings/#/integrations/native_apps?integrationIframeTarget=integrations/intacct".format(settings.FYLE_APP_URL)
-                        }
+                        'app_url': "{0}/app/settings/#/integrations/native_apps?integrationIframeTarget=integrations/intacct".format(settings.FYLE_APP_URL),
+                        'fyle_url': settings.FYLE_EXPENSE_URL,
```
Comment (ashwin1111) [inline apps/workspaces/tasks.py:180]: not needed anymore

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**